src/components/cookiebanner.js

```diff
@@ -0,0 +1,85 @@
+import * as React from "react"
+import { Container, Row, Col, Button } from "react-bootstrap"
+import { withCookies } from 'react-cookie'
+
+const CookieBanner = ({ cookies }) => {    
+    const setCookies = (cookiesBool) => {
+        cookies.set('gatsby-gdpr-google-tagmanager', cookiesBool) // so named to allow
+    }
+
+    const onKeyDown = (event, cookies) => {
+        switch (event.keyCode) {
+            case 13:
+                setCookies(cookies);
+                event.stopPropagation();
+                break
+            default:
+                return
+        }
+    }
+
+    return (typeof cookies.get('gatsby-gdpr-google-tagmanager') == 'undefined') ? (
+        <Container
+            fluid
+            style={{
+                background: `#e1dcf2`,
+                position: `fixed`,
+                bottom: 0,
+                paddingTop: `10px`,
+                paddingBottom:`10px`
+            }}
+        >
+            <Row
+                className="justify-content-end"
+                style={{
+                    textAlign: "right",
+                }}
+            >
+                <Col xs="8"
+                    className="d-inline"
+                    style={{
+                        textAlign: "left",
+                    }}
+                >
+                    We use cookies and analytics to track your usage of our site. You may opt-out by clicking "reject", or opt-in by clicking "accept".
```

Comment:
Is there a way to add some sort of border to the footer (or alternatively, use a slightly different bg color) so that the text stands out more? Right now it's easy to miss since it blends right into the bg of the landing page - I only noticed it b/c I was explicitly looking for it:

<img width="1512" alt="Screen Shot 2022-05-02 at 5 54 44 PM" src="https://user-images.githubusercontent.com/2418071/166333901-ed5f25a6-90f9-4ef6-91c5-041a4a8ce717.png">

Reply:
yeah sec I'll make it bold / white on a deep purple

Reply:
![image](https://user-images.githubusercontent.com/7660110/166466776-ea03fbd9-3dee-4670-89b8-c2a005719ab9.png)

Reply:
oooo dat nice

Reply:
Got a chance to build and test it locally - much easier to notice now 👍  ship it yeet it